### PR TITLE
Add LimitPolicy metrics to Prometheus

### PR DESCRIPTION
**Pull request by bschimke95**
Integrates the internal metrics of the `limit` policy with Prometheus

### CODE REVIEW

**Review by neoaggelos — COMMENTED** [review]
Nice work. I would recommend one final change. In the existing metrics, we use the terms `operation` and `transaction`. `operation` is the intent (e.g. add a new record), and to fulfill an operation we might retry multiple `transactions`.

In this case, the new metrics are closely related to `operations` instead of individual `transactions`. Can we change the name of the metrics to reflect this?

File: pkg/kine/drivers/generic/metrics.go [hunk 1]
```diff
@@ -20,6 +20,14 @@ var (
 		Help:    "Transaction latency of database operations by tx_name and result",
 		Buckets: []float64{0, 0.05, 0.1, 0.3, 0.5, 1, 3, 5, 10},
 	}, []string{"tx_name", "result"})
+	metricsCurrentOps = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "k8s_dqlite_generic_current_ops",
+		Help: "Number of database operations that run concurrently by tx_name",
```
Comment (neoaggelos) [inline pkg/kine/drivers/generic/metrics.go:25]:
```suggestion
		Help: "Total number of database operations that are currently running by tx_name",
```

File: pkg/kine/drivers/generic/metrics.go [hunk 2]
```diff
@@ -20,6 +20,14 @@ var (
...
+	}, []string{"tx_name"})
+	metricsOpAdmissionControl = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "k8s_dqlite_generic_op_admission_control",
+		Help: "Number of database operations that the admission control handled by by tx_name and status",
+	}, []string{"tx_name", "status"})
```
Comment (neoaggelos) [inline pkg/kine/drivers/generic/metrics.go:30]:
```suggestion
	}, []string{"tx_name", "result"})
```

**Review by neoaggelos — APPROVED** [review]
LGTM, left some minor wording suggestions

**Comment by ktsakalozos** [comment]
LGTM +1

**Comment by bschimke95** [comment]
This PR was targeting #69 instead of master. #73 targets the correct branch.

**Pull request merged**